gui/_global.scss

```diff
@@ -29,6 +29,14 @@ h4 {
   font-size: 1.5rem;
 }
 
+a {
+  text-decoration: none;
+}
+
+a:hover {
+  text-decoration: underline;
+}
+
```

Comment:
There's a hyperlink rule available with `_typography.scss`. Could you please help move this over there? Thanks.

Reply:
Okay, I'm on it.